Makefile

```diff
@@ -0,0 +1,17 @@
+clean:
+	@rm -rf `find . -name __pycache__`
+	@rm -f `find . -type f -name '*.py[co]' `
```

Comment:
Please. `find` has a `-delete` flag.

Reply:
But it doesn't delete folders.
Let's keep it to be mimic previous line.
Or, most likely, we could delete the line at all -- every `pyc` file sits in `__pycache__` anyway.

Reply:
It will if you match by type `d`.

Reply:
Delete non-empty directory? Are you sure?

Reply:
Nope, but just swap these rules: first delete files, then directories.